<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui::RunScriptWidget</class>
 <widget class="QWidget" name="Gui::RunScriptWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1266</width>
    <height>44</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Run Script Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,0,0,0,0,0">
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="2">
    <widget class="QToolButton" name="openButton">
     <property name="toolTip">
      <string>Select Script</string>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextOnly</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit">
     <property name="placeholderText">
      <string>Name or path of a script</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QToolButton" name="closeButton">
     <property name="text">
      <string>&amp;Close</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextOnly</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Script:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QToolButton" name="runButton">
     <property name="text">
      <string>&amp;Run</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextOnly</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots/>
</ui>
